packages/fast-components-react-base/src/toolbar/README.md

```diff
@@ -0,0 +1,24 @@
+## Toolbar
+A generic toolbar component implementing the [toolbar](https://www.w3.org/TR/wai-aria-1.1/#toolbar) role and the keyboard navigation aligning to that role. The *toolbar* will accept any child elements but will only assign focus to children with roles specified in the component's "focusableRoles" prop.
+
+### Usage
+To accurately manage document focus, the *toolbar* assumes that all only children with the appropriate roles should be focusable. It also expects that these elements are passed as *direct* children. Wrapping all menu items inside of a `React.Fragment` (or some other single element) will throw off the internal focus tracking of the *toolbar*. Additionally, the *toolbar* expects that all menu item children implement the `tabIndex` property and apply that property to the underlying DOM.
```

Comment:
This seems to imply that the children of a toolbar are always menu items of some kind. Toolbar is fundamentally different than a menubar - have these been convoluted?

Reply:
corrected